homeworks/vulmanjanos/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,19 @@
+character_first_name = input("Please give me your first name:")
+character_family_name = input("Give me your family name too:")
+character_full_name = character_first_name + character_family_name
+character_age = input("How old are you?")
+character_python_experience = input("How many years Phyton experience you have?")
+
+character_first_name = character_first_name.strip().capitalize()
```

Comment:
nem hiba, csak megjegyzés:
ha akarod, ezeket meghívhatod az első soron is:
`character_first_name = input("Please give me your first name:").strip().capitalize()`
ugyanez igaz a character_family_name-re is.